.github/workflows/ci.yml

```diff
@@ -260,3 +260,77 @@ jobs:
       # when imported in an NPM node_modules tree.
       - name: Prepack packages
         run: yarn lerna run --reject-cycles --concurrency 1 prepack
+
+  test-xs:
+    name: test-xs
+
+    # begin macro
+
+    runs-on: ${{ matrix.platform }}
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [20.x]
+        # may include release tags and hashes:
+        moddable-version: [5.0.0]
+        platform: [ubuntu-latest]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      # without this, setup-node errors on mismatched yarn versions
+      - run: corepack enable
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: yarn
+
+      - name: Install dependencies
+        run: yarn install --immutable
+
+      # end macro
+
+      - name: Run yarn build
+        run: yarn build
+
+      - name: Choose download or build
+        id: check-release
+        run: |
+          if curl -f -s -o /dev/null -I -L https://api.github.com/repos/Moddable-OpenSource/moddable/releases/tags/${{ matrix.moddable-version }}
+          then
+            echo release=download
+          else
+            echo release=build
+          fi >> $GITHUB_OUTPUT
+
+      - name: Download XS
+        if: steps.check-release.outputs.release == 'download'
+        working-directory: ..
+        run: |
+          BINDIR=$GITHUB_WORKSPACE/moddable/build/bin/lin/debug
+          wget "https://github.com/Moddable-OpenSource/moddable/releases/download/${{ matrix.moddable-version }}/xst-lin64.zip"
+          mkdir -p "$BINDIR"
+          unzip xst-lin64.zip -d "$BINDIR"
+          chmod 755 "$BINDIR/xst"
+
+      - name: Checkout XS
+        if: steps.check-release.outputs.release == 'build'
+        uses: actions/checkout@v4
+        with:
+          repository: moddable-OpenSource/moddable
+          ref: ${{ matrix.moddable-version }}
+          path: moddable
+
+      - name: Build XS
+        if: steps.check-release.outputs.release == 'build'
+        working-directory: moddable/xs/makefiles/lin
+        run: |
+          make debug MODDABLE=$GITHUB_WORKSPACE/moddable CC='cc "-D__has_builtin(x)=1"' # give the syntax highlighter a hand: '
```

Comment:
```suggestion
          make debug MODDABLE=$GITHUB_WORKSPACE/moddable CC='cc "-D__has_builtin(x)=1"' # here's a nickle, kid... go get yourself a better syntax highlighter
```

Reply:
And that’s the difference between giving your 2¢ and giving your 5¢.